dts/arm/gigadevice/gd32f4xx/gd32f450ik.dtsi

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2021 Teslabs Engineering S.L.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <mem.h>
+#include <gigadevice/gd32f4xx/gd32f4xx.dtsi>
+
+/ {
+	soc {
+		flash-controller@40023c00 {
+			flash0: flash@8000000 {
+				reg = <0x08000000 DT_SIZE_K(3072)>;
+			};
+		};
+
+		sram0: memory@20000000 {
+			reg = <0x20000000 DT_SIZE_K(112)>;
+		};
```

Comment:
This is same for all 4xx series and can be defined at gd32f4xx.dtsi. Only SRAM2 and ADDSRAM should be defined here for f450.

Reply:
done